Doc/library/tkinter.font.rst

```diff
@@ -91,6 +91,9 @@ The different font weights and slants are:
 
    Return the names of defined fonts.
 
-.. function:: nametofont(name)
+.. function:: nametofont(name, root=None)
 
-   Return a :class:`Font` representation of a tk named font.
\ No newline at end of file
+   Return a :class:`Font` representation of a tk named font.
+
+  .. versionchanged:: 3.10
```

Comment:
Seems to be missing a space here, indentation in rST is 3 spaces. On some versions of sphinx this causes visual differences.

```suggestion
   .. versionchanged:: 3.10
      Added the *root* keyword argument.   
```
Other than that, LGTM :).

Reply:
Thanks for catching that! I've updated the documentation accordingly